extension/app/ts/background/windows/personalSign.ts

```diff
@@ -43,6 +43,81 @@ function reject() {
 	}
 }
 
+export function craftPersonalSignPopupMessage(params: PersonalSignParams | SignTypedDataParams, activeAddress: bigint, userAddressBook: UserAddressBook, simulationMode: boolean, requestId: number) {
+	if (params.method === 'personal_sign') {
+		return {
+			method: 'popup_personal_sign_request',
+			data: {
+				activeAddress,
+				type: 'NotParsed' as const,
+				simulationMode: simulationMode,
+				requestId: requestId,
+				message: params.params[0],
+				account: getAddressMetaData(params.params[1], userAddressBook),
+				method: params.method,
+				params: params
```

Comment:
If you have a local parameter with the same name as the property, you can just list the local without the property name.  In fact, you already do this for `activeAddress`.
```suggestion
				activeAddress,
				type: 'NotParsed' as const,
				simulationMode,
				requestId,
				message: params.params[0],
				account: getAddressMetaData(params.params[1], userAddressBook),
				method: params.method,
				params,
```